app/controllers/two_factor_authentication/options_controller.rb

```diff
@@ -58,6 +58,11 @@ def two_factor_options_presenter
     end
 
     def process_valid_form
+      analytics.multi_factor_auth_method_selected(
```

Comment:
I'm just now noticing the `multi_factor_auth_option_list` above. Is that the event we're wanting with this ticket? 🤔

Reply:
It looks like it.

Reply:
It is. I wonder if this event should go inside the `create` event, or after valid submission. Either way, I need to add the `enabled_mfa_methods_count` and mfa_methods_count` to that event for this ticket.

Reply:
Now that I think about it some more, I will keep it solely in the `#create` action